modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -73,4 +80,23 @@ public SELF withPassword(final String password) {
         this.password = password;
         return self();
     }
+
+    @Override
+    protected void waitUntilContainerStarted() {
+        String regEx = ".*database system is ready to accept connections.*\\s";
+        logger().info("Waiting for database to log '{}'", regEx);
+
+        WaitingConsumer waitingConsumer = new WaitingConsumer();
+        this.followOutput(waitingConsumer);
+
+
+        Predicate<OutputFrame> waitPredicate = outputFrame ->
+                outputFrame.getUtf8String().matches(regEx);
```

Comment:
Good point, this means we should fix this in `LogMessageWaitStrategy` as well I assume?
https://github.com/testcontainers/testcontainers-java/blob/c2eae85232653273748aaa4f5f765f48c28e0f87/core/src/main/java/org/testcontainers/containers/wait/LogMessageWaitStrategy.java#L25